iXBRLViewerPlugin/viewer/src/js/report.js

```diff
@@ -266,4 +266,17 @@ export class XBRLReport {
         }
         return this._calculationSummations.has(c);
     }
+
+
+    /**
+     * @return {Array[String]} Software credit text labels provided with this report for display purposes.
+     */
+    softwareCredits() {
+        if (!this._reportData.softwareCredits) {
+            return [];
+        }
+        // Arelle defaults to "None" if no comment text is found. This will be null for our purposes.
+        return this._reportData.softwareCredits
+                .filter(c => c !== "None");
```

Comment:
It's unfortunate that the model returns `'None'` rather than `None`.  In any case, it seems like this should be filtered out in the Python instead.

Reply:
I suppose with the Arelle `creationSoftwareMatches` refactor I did before this ticket, this filter is no longer necessary anywhere (the main point of the refactor was to decouple the software name matching from undesired existing behavior of returning "None" or spitting out the full comment text)